data/net/minecraft/client/gui/GuiGraphics.mapping

```diff
@@ -7,240 +7,487 @@ CLASS net/minecraft/client/gui/GuiGraphics
+			COMMENT The x-coordinate of the second corner of the blit position.
 		ARG 4 y1
+			COMMENT The y-coordinate of the first corner of the blit position.
 		ARG 5 y2
+			COMMENT The y-coordinate of the second corner of the blit position.
 		ARG 6 blitOffset
+			COMMENT The z-level offset for rendering order.
 		ARG 7 uWidth
+			COMMENT The width of the blitted portion in texture coordinates.
 		ARG 8 vHeight
+			COMMENT The height of the blitted portion in texture coordinates.
 		ARG 9 uOffset
+			COMMENT The horizontal texture coordinate offset.
 		ARG 10 vOffset
+			COMMENT The vertical texture coordinate offset.
 		ARG 11 textureWidth
+			COMMENT The width of the texture.
 		ARG 12 textureHeight
+			COMMENT The height of the texture.
 	METHOD blitNineSliced (Lnet/minecraft/resources/ResourceLocation;IIIIIIIII)V
+		COMMENT Blits a nine-sliced portion of the texture specified by the atlas location onto the screen.
 		ARG 1 atlasLocation
-		ARG 2 targetX
-		ARG 3 targetY
-		ARG 4 targetWidth
-		ARG 5 targetHeight
+			COMMENT The location of the texture atlas.
+		ARG 2 x
+			COMMENT The x-coordinate of the top-left corner of the target position.
+		ARG 3 y
+			COMMENT The y-coordinate of the top-left corner of the target position.
+		ARG 4 width
+			COMMENT The width of the target portion.
+		ARG 5 height
+			COMMENT The height of the target portion.
 		ARG 6 sliceSize
-		ARG 7 sourceWidth
-		ARG 8 sourceHeight
-		ARG 9 sourceX
-		ARG 10 sourceY
+			COMMENT The size of the corner slices.
+		ARG 7 uOffset
+			COMMENT The width of the source texture.
+		ARG 8 vOffset
+			COMMENT The height of the source texture.
+		ARG 9 textureWidth
+			COMMENT The x-coordinate of the top-left corner of the source position.
+		ARG 10 textureHeight
+			COMMENT The y-coordinate of the top-left corner of the source position.
 	METHOD blitNineSliced (Lnet/minecraft/resources/ResourceLocation;IIIIIIIIII)V
+		COMMENT Blits a nine-sliced portion of the texture specified by the atlas location onto the screen.
 		ARG 1 atlasLocation
-		ARG 2 targetX
-		ARG 3 targetY
-		ARG 4 targetWidth
-		ARG 5 targetHeight
+			COMMENT The location of the texture atlas.
+		ARG 2 x
+			COMMENT The x-coordinate of the top-left corner of the target position.
+		ARG 3 y
+			COMMENT The y-coordinate of the top-left corner of the target position.
+		ARG 4 width
+			COMMENT The width of the target portion.
+		ARG 5 height
+			COMMENT The height of the target portion.
 		ARG 6 sliceWidth
+			COMMENT The width of the corner slices.
 		ARG 7 sliceHeight
-		ARG 8 sourceWidth
-		ARG 9 sourceHeight
-		ARG 10 sourceX
-		ARG 11 sourceY
+			COMMENT The height of the corner slices.
+		ARG 8 uWidth
+			COMMENT The width of the source texture.
+		ARG 9 vHeight
+			COMMENT The height of the source texture.
+		ARG 10 textureWidth
+			COMMENT The x-coordinate of the top-left corner of the source position.
+		ARG 11 textureHeight
+			COMMENT The y-coordinate of the top-left corner of the source position.
```

Comment:
Just as above, the Javadoc on these parameters seems inconsistent with their name. `textureWidth` should not represent an x-coordinate.